docs/packaging-a-RNComponent-ios.md

```diff
@@ -0,0 +1,77 @@
+---
+id: packaging-a-RNComponent-ios
+title: Packaging a react-native component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-a-RNComponent-ios.html
+next: native-modules-android
+---
+
+After you've written some code in an existing project and it works great
+and you want to pull it out into its own package, put it on npm, then
+install it back into your project. But wait, your component is partially
+implemented in Objective-C! How can you share this on `npm`? 
+Please read the following steps
+### Creating the package
+
+1. Create a new project, use this type of naming, RN prefix.
```

Comment:
We could probably link here to a document describing namespacing issues and why that's important? `RN` is good, however it may produce some conflicts, I usually name it with the domain, e.g. `MGR` (which is my first name and last name shortcut)